lib/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@radix-ui/react-popover": "^1.0.7",
+    "@radix-ui/react-tooltip": "^1.0.7",
```

Comment:
I tried implementing it with the same react-popover to prevent adding an extra library. However, the interaction with the current modals is buggy that way (dropdown, calendar...)